src/reporting/reportingToken.se

```diff
@@ -98,24 +115,39 @@ def redeemWinningTokens(reporter: address):
     reporterReputationShare = reputationSupply * attotokens / self.supply
     self.supply -= attotokens
     self.tokens[reporter] = 0
-    self.getReputationToken().transfer(reporter, reporterReputationShare)
-    log(type=Transfer, reporter, 0, attotokens)
+    if (reporterReputationShare > 0):
+        self.getReputationToken().transfer(reporter, reporterReputationShare)
+        log(type=Transfer, reporter, 0, attotokens)
     return 1
 
 # FIXME: bond holders have already been paid at this point (they are paid as soon as market finalization occurs)
 # CONSIDER: track how much bond holders are owed, then as REP is extracted from losing tokens it initially goes into
 def extractBondHolderPayments():
```

Comment:
Do we still need this function?  I believe now the mechanic is to extract REP from losing tokens during the losing token migration process, rather than at `redeemWinningToken` time so attempting to extract them again here is wasteful since it will never do anything.

This would explain why the `getDesiredDisputeRewardPot` function is not causing test failures despite the undefined constants.